app/views/provider_interface/reports/diversity_reports/show.html.erb

```diff
@@ -0,0 +1,94 @@
+<%= content_for :title, t('page_titles.provider.diversity_report') %>
+<%= content_for :before_content, breadcrumbs(t('page_titles.provider.reports') => provider_interface_reports_path,
+                                             t('page_titles.provider.diversity_report') => nil) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-l">
+      <span class="govuk-caption-l"> <%= @provider.name %></span>
+      <%= t('page_titles.provider.diversity_report') %>
+    </h1>
+  </div>
+</div>
+
+<% if @completed_e_and_d_survey_count < 10 %>
+  <p class="govuk-body">
+    You’ll be able to see this report once it contains data from at least 10 candidates. This is to protect the privacy of candidates.
+  </p>
+<% else %>
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+      <p class="govuk-body">
+        <%= govuk_link_to 'Export data (ZIP)', provider_interface_reports_provider_diversity_report_path(provider_id: @provider, format: :zip) %>
+      </p>
+      <h2 class="govuk-heading-m">Sex</h3>
+      <%= render ProviderInterface::ReportTableComponent.new(headers: ['Sex',
+                                                                       'Applied',
+                                                                       'Offered',
+                                                                       'Recruited',
+                                                                       'Percentage recruited'],
```

Comment:
Can we move to one of the classes created in the PR

Reply:
`headers: ['Disability or health condition', *ProviderInterface::DiversityDataByProvider::REPORT_HEADERS]` etc.